specification/inference/delete/DeleteRequest.ts

```diff
@@ -33,9 +33,22 @@ export interface Request extends RequestBase {
      * The task type
      */
     task_type?: TaskType
+
     /**
      * The inference Id
      */
     inference_id: Id
+
```

Comment:
```suggestion
query_parameters: {
```

These should be in query_parameters rather than path parts